server/app/auth/oidc/OidcProfileAdapter.java

```diff
@@ -44,9 +44,9 @@ public OidcProfileAdapter(
       OidcClient client,
       ProfileFactory profileFactory,
       Provider<UserRepository> applicantRepositoryProvider) {
-    super(configuration, client);
+    super(Preconditions.checkNotNull(configuration), Preconditions.checkNotNull(client));
```

Comment:
Not blocking: We don't make use of the configuration / client below and they're purely a concern of the base class. What do you think about removing the checkNotNull since it's the responsibility of the base class to check that?